unpackaged/post/facility_display_name/objects/___NAMESPACE___Facility__c.object

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fields>
+        <fullName>Display_Name__c</fullName>
+        <description>The full name of the Facility. For example, &quot;Thomas Edison Building - Room 201.&quot;</description>
+        <externalId>false</externalId>
+        <formula>IF(
+    NOT(ISBLANK(Parent_Facility__c)), Parent_Facility__r.Name + &apos; - &apos; + Name,
+    IF(
+        NOT(ISBLANK(Account__c)), Account__r.Name + &apos; - &apos; + Name, 
+        Name
+    )
+)</formula>
+        <formulaTreatBlanksAs>BlankAsZero</formulaTreatBlanksAs>
```

Comment:
@robertbwatson -- I noticed this is blanks as zero, in my experience the platform does the right thing when working with text and this set as such, but just thought I'd ask if we want to consider "blanks as null." I believe we are fine either way.

Reply:
@lifewithryan Oh, weird. I'm going to keep this as-is, but good catch. I see the "Relationship Explanation" field on the Relationship object is the same. There's not actually a way to change this value in the UI, so this is what I got back from Salesforce when doing a retrieve.